tests/fe/fe_conformity_dim_2_fe_nedelec.cc

```diff
@@ -42,7 +42,7 @@ main(int, char **)
 
       constexpr int dim = 2;
 
-      for (unsigned int fe_degree = 0; fe_degree < 5; ++fe_degree)
+      for (unsigned int fe_degree = 0; fe_degree < 1; ++fe_degree)
```

Comment:
Could we add a comment like https://github.com/dealii/dealii/pull/12016#issuecomment-817137541 here to remind ourselves that this is an issue?